### PR TITLE
Bug 1779348: [release-4.3] vendor: bring in ReservedSystemCPUs changes

### PR DESCRIPTION
**Pull request by runcom**
Manual backport of #1331 

Signed-off-by: Antonio Murdaca <runcom@linux.com>

### CODE REVIEW

**Comment by vladikr** [comment]
:+1: look good to me!

File: vendor/k8s.io/kubelet/config/v1beta1/types.go [hunk 1]
```diff
@@ -710,6 +710,10 @@ type KubeletConfiguration struct {
 	// Default: nil
 	// +optional
 	KubeReserved map[string]string `json:"kubeReserved,omitempty"`
+	// This ReservedSystemCPUs option specifies the cpu list reserved for the host level system threads and kubernetes related threads.
+	// This provide a "static" CPU list rather than the "dynamic" list by system-reserved and kube-reserved.
+	// This option overwrites CPUs provided by system-reserved and kube-reserved.
+	ReservedSystemCPUs string `json:"reservedSystemCPUs,omitempty"`
```
Comment (sinnykumari) [inline vendor/k8s.io/kubelet/config/v1beta1/types.go]: +1

**Comment by sinnykumari** [comment]
Thanks Antonio!

**Pull request merged**